department-of-veterans-affairs/va.gov-cms
# VACMS-4520 4523 Disable title and display intro

## Description
See #4520 #4523

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/109812403-045a4f80-7bfa-11eb-9ad5-2ceb9060b039.png)
![image](https://user-images.githubusercontent.com/2404547/109812417-08866d00-7bfa-11eb-825a-42cab381e92d.png)

## QA steps
- As an administrator, go to `node/2360/edit` and visually verify that you can alter the title.
- Logout, login back in as a user without administrator role, and visually verify that the title field is no longer editable.
- As an administrator, go to `/health-care/covid-19-vaccine-tag` and visually verify that the page introduction is visible.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
